.NET/Microsoft.Recognizers.Text.NumberWithUnit/Parsers/CurrencyUnitValue.cs

```diff
@@ -1,7 +1,19 @@
-﻿namespace Microsoft.Recognizers.Text.NumberWithUnit
+﻿using System.Collections.Generic;
+
+using Microsoft.Recognizers.Text.NumberWithUnit.Utilities;
+
+namespace Microsoft.Recognizers.Text.NumberWithUnit
 {
-    public class CurrencyUnitValue : UnitValue
+    public class CurrencyUnitValue
```

Comment:
This class inherited from UnitValue. Why it does not anymore?

Reply:
It was a bad merge, fixed